packages/airnode-deployer/src/utils/infrastructure.ts

```diff
@@ -163,3 +167,38 @@ export const deploymentComparator = (a: Deployment, b: Deployment) => {
 
   return compareVersions(a.airnodeVersion, b.airnodeVersion);
 };
+
+export const checkBucketMissingFiles = (
```

Comment:
Let's call it `getMissingBucketFiles` as it returns something rather than just checking and potentially throwing an exception.